docs/platform-and-architecture/marlowe-language-guide.md

```diff
@@ -158,4 +158,36 @@ The outermost `When` calls for the first choice to be made by Alice: if Alice ha
 A Marlowe account holds amounts of multiple currencies and/or fungible and non-fungible tokens. A concrete amount is indexed by a `Token`, which is a pair of `CurrencySymbol` and `TokenName`. You can think of an account as a map token integer, where:
 `data Token = Token CurrencySymbol TokenName`
 
-The ada token of Cardano is represented as `Token adaSymbol adaToken`, however, you can create your own currencies and tokens. 
\ No newline at end of file
+The ada token of Cardano is represented as `Token adaSymbol adaToken`, however, you can create your own currencies and tokens. 
+
+## Minimum ada requirements
+
+There is a minimum ada requirement to create a contract. However, when the contract involves different native tokens in the output, the minimum ada requirement increases to accommodate every potential contract path. 
+
+### Ledger rule
+
+The **[minimum ada requirement](https://docs.cardano.org/native-tokens/minimum-ada-value-requirement/)** is a ledger rule that requires every Tx output to contain a minimum amount of ada. 
+
+This amount increases with the number of different native tokens in the output. The node will reject transaction outputs that don't satisfy the minimum ada requirement.
+
+### Deposited at contract creation, refunded at closing
+
+The minimum ada deposit is deposited into a Marlowe contract account when it is created. The deposit is held in this account until the contract closes. Afterwards it is refunded. 
+
+### Maximum required ada deposit
+
+The amount of ada deposited should be enough to cover the maximum required ada deposit for the Marlowe script output at any point in the contract.
```

Comment:
```suggestion
The amount of ada deposited should be enough to cover the minimum required ada deposit for the Marlowe script output at any point in the contract.
```

We're talking about the *maximum* of all of the *minimums* here, which is a source of potential confusion.